tide/master/rest/RestCommand.cpp

```diff
@@ -62,6 +88,13 @@ bool RestCommand::_fromJSON( const std::string& string )
         put_flog( LOG_INFO, "Error parsing JSON string: '%s'", string.c_str( ));
         return false;
     }
+
+    if( !_takesValue )
```

Comment:
can't you check if doc.object() has 'uri' key value? Doesn't need to be hardcoded like this I think.